app/forms/claimant_form.rb

```diff
@@ -76,6 +77,10 @@ def last_name=(name)
 
   private
 
+  def other_title_selected?
+    title == I18n.t("claims.personal_details.title.options.Other")
```

Comment:
Have you tested this in welsh ?  I don't think it will work.

The dropdown should be configured so that the key from the translation file is what is stored in the database, not the value shown to the user which is the value from the translation file

If you look, the keys are the same irrespective of language - but your code will translate to 'Arall' which will return false